src/panels/lovelace/components/hui-generic-entity-row.ts

```diff
@@ -112,6 +112,16 @@ class HuiGenericEntityRow extends LitElement {
                     : this.hass.localize(
                         "ui.panel.lovelace.cards.entities.never_triggered"
                       )
+                  : this.config.secondary_info === "position" &&
+                    stateObj.attributes.current_position
```

Comment:
`current_position` can be 0 I think? So we shouldn't do a falsy check, but check for `undefined`

Same for tilt.

Reply:
Good point. What do you think about just omitting the check itself? It would then be `Position: undefined` for not supported covers. This way you would instantly know that your ui-config is wrong.

I'll push the undefined check in a few minutes. Let me know if you should remove them.

Reply:
it won't print undefined